torch/quantization/quantize.py

```diff
@@ -88,17 +88,17 @@ def add_observer(module):
             forward_hooks
     """
     for child in module.children():
-        add_observer(child)
+        if type(child) == nnq.FloatFunctional:
```

Comment:
what? are you going to insert two observer modules in this case?

Reply:
No, it's conditional on if the child is a leaf